### PR TITLE
Update LICENSE to the common template one

### PR DESCRIPTION
**Pull request by dominykas**
See https://github.com/nodejs/package-maintenance/issues/348

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,7 +1,6 @@
+MIT License
 
-The MIT License (MIT)
-
-Copyright (c) 2019 Wes Todd <wes@wesleytodd.com>
+Copyright Node.js contributors. All rights reserved.
```
Comment (dominykas) [inline LICENSE:3]: @wesleytodd is this OK with you?

We still have to decide what we're going to do about the `AUTHORS` file and whether to link to it here, but that's a dicussion for https://github.com/nodejs/package-maintenance/issues/348.

**Comment by wesleytodd** [comment]
FWIW, I think we should deprecate and remove this repo.  I created this before the org moved into node/pm wg ownership as I was building the status board.  This really isn't all that helpful in this context, I only put it here because I did it at the same time as the statusboard and used it in conjunction with building it.  Maybe I should just move it to my account?

**Comment by dominykas** [comment]
> Maybe I should just move it to my account?

Is our charter official yet? Do we need a vote? 😂 

I don't mind if this gets removed.

**Comment by dominykas** [comment]
OK, closing this PR in favor of https://github.com/nodejs/package-maintenance/issues/384

**Pull request closed without merging**